package.json

```diff
@@ -81,6 +81,7 @@
     "glossary": "^0.1.1",
     "google-spreadsheet": "^2.0.3",
     "graphql": "^0.7.2",
+    "immutable": "^3.8.2",
```

Comment:
It doesn't seem like we benefit from Immutable enough to justify the dependency. I'd consider removing it if that wasn't a huge investment.

Reply:
We need to use it for the `draft-js` library, in order to pass custom blocks to the editor -- there are a few more places it comes up in a follow-up to this pr.